azure/database/postgres/variables.tf

```diff
@@ -0,0 +1,118 @@
+variable "name" {
```

Comment:
should we add a setting for `high_availability`? See https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/postgresql_flexible_server#high_availability